src/shapes/IText/IText.ts

```diff
@@ -204,10 +204,6 @@ export class IText<
 
   static type = 'IText';
 
-  get type() {
-    return 'i-text';
-  }
```

Comment:
this should stay.

Reply:
But is breaks subclassing
Textbox is broken

Reply:
We should instead override isType like you did with object

Reply:
Fixed like you did with object